### PR TITLE
Address few existing documentation issues

### PR DESCRIPTION
**Pull request by nmav**

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -122,5 +122,10 @@ upload-release: $(DIST_ARCHIVES)
 	gpg --detach-sign --local-user 'stef@thewalter.net' $<
 	scp $< $<.sig $(WEBHOST):$(WEBBASE)/releases/
 
+README:
+	cp README.md README
+
 transifex:
 	cd $(srcdir) && sh build/tx-update
+
+.PHONY: README
```
Comment (ueno) [inline Makefile.am]: I don't think `.PHONY` is appropriate here; it's for non-filename target as far as I understand it.

File: Makefile.am [hunk 2]
```diff
@@ -122,5 +122,10 @@ upload-release: $(DIST_ARCHIVES)
 	gpg --detach-sign --local-user 'stef@thewalter.net' $<
 	scp $< $<.sig $(WEBHOST):$(WEBBASE)/releases/
 
+README:
+	cp README.md README
```
Comment (ueno) [inline Makefile.am]: Maybe this could be done as part of `dist-hook` defined above?

File: Makefile.am (hunk 1 above)
Comment (nmav) [reply Makefile.am]: removed

**Comment by ueno** [comment]
Thank you!

**Pull request merged**